reconcile/utils/terrascript_aws_client.py

```diff
@@ -2451,6 +2464,40 @@ def populate_tf_resource_s3_cloudfront(self, resource, namespace_info):
             '${' + cf_oai_tf_resource.id + '}'
         tf_resources.append(Output(output_name_0_13, value=output_value))
 
+        # aws_s3_bucket_acl
+        values = common_values.get('distribution_config', {})
+        if 'logging_config' in values.keys():
+            logging_config_bucket = values['logging_config']
+            values = {}
+            access_control_policy = {
+                'owner': {
+                    'id': '${data.aws_canonical_user_id.current.id}',
+                },
+                'grant': [
+                    {
+                        'grantee': {
+                            'id': '${data.aws_canonical_user_id.current.id}',
+                            'type': 'CanonicalUser',
+                        },
+                        'permission': 'FULL_CONTROL',
+                    },
+                    {
+                        'grantee': {
+                            # well known id for cloudfront logs delivery:
+                            # https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/AccessLogs.html#AccessLogsBucketAndFileOwnership
+                            'id': 'c4c1ede66af53448b93c283ce9448c4ba468c9432aa01d700d3878632f77d2d0',
+                            'type': 'CanonicalUser',
+                        },
+                        'permission': 'FULL_CONTROL',
```

Comment:
Is this really needed? 
I understand the ACL is set to the bucket when the CF distribution is created. I think we are using the same user to create the bucket and the distribution so, it won't be enough with the other grant?

Reply:
If we don't set it here, it might be added by CF (not sure with which permissions, it's unclear to me) but will then be removed by terraform runs, right ?

Reply:
That makes sense